generators/app/templates/src/index.js

```diff
@@ -0,0 +1,51 @@
+export default {
+    // Multiple browsers support
+    isMultiBrowser: <%= isMultiBrowser %>,
+
+
+    // Required - must be implemented
+    // Browser control
+    async openBrowser (/* id, browserName, startPage */) {
+        throw new Error('Not implemented!');
+    },
+
+    async closeBrowser (/* id, pageInfo */) {
```

Comment:
and pls don't forget to keep it in sync with `phantomjs` and `saucelabs` providers - so that we don't have to say it twice :gun: (like @MargaritaLoseva had to)